mitmproxy/proxy/protocol/http.py

```diff
@@ -327,7 +330,15 @@ def _process_flow(self, f):
                 )
 
                 def get_response():
```

Comment:
As discussed on Slack, this is specific to this PR - `get_response` is retried, and on retry we have potentially already streamed some parts of the body.